topi/python/topi/cuda/dense.py

```diff
@@ -23,54 +23,16 @@
 from tvm.autotvm.task.space import SplitEntity
 from tvm.contrib import cublas
 from .tensor_intrin import dp4a
-from ..nn.dense import dense, dense_default
+from ..nn import dense
```

Comment:
```suggestion
from .. import nn
```
Let just use `nn.dense` to avoid potential duplicate naming